test/expected/tablespace.out

```diff
@@ -340,14 +395,147 @@ SELECT * FROM _timescaledb_catalog.tablespace;
 ----+---------------+-----------------
 (0 rows)
 
+-- Create two tables and attach multiple tablespaces to them. Verify
+-- that dropping a tablespace from multiple tables work as expected.
+CREATE TABLE tspace_1dim(time timestamp, temp float, device text);
+SELECT create_hypertable('tspace_1dim', 'time');
+NOTICE:  adding not-null constraint to column "time"
+    create_hypertable     
+--------------------------
+ (3,public,tspace_1dim,t)
+(1 row)
+
+CREATE TABLE tspace_2dim(time timestamp, temp float, device text);
+SELECT create_hypertable('tspace_2dim', 'time');
```

Comment:
This is a confusing choice of name. Previously, 1dim, 2dim, etc. indicated the number of dimensions of the table but here all tables are created with one dimension.

The reason why you want to test tablespaces on, e.g., a space partitioned table is because tablespaces should be sticky to space partition.

Reply:
Switched to `tbl_1`, `tbl_2`, and `tbl_3`.